File: EmployeeDataAccessLibrary/Models/Person.cs
using System.Collections.Generic;

namespace EmployeeDataAccessLibrary.Models
{
    public class Person
    {
        public int Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public int Age { get; set; }
        public List<Address> Addresses { get; set; } = new List<Address>();
        public List<Email> EmailAddresses { get; set; } = new List<Email>();
    }
}

File: AngularFolder/EmpReviewer/src/app/models/person.ts

export class Person {
    id:number;
    firstName:string;
    lastName:string;
    age:number;
    addresses: Address[];
    emailAddresses: EmailAddress[];
}

export class EmailAddress {
    id:number;
    emailAddress:string;
}

export class Address {
    id:number;
    streetAddress: string;
    city: string;
    state: string;
    zipCode: string;
}
